src/platform/plugins/shared/discover/public/embeddable/actions/view_saved_search_action.ts

```diff
@@ -48,7 +48,7 @@ export class ViewSavedSearchAction implements Action<EmbeddableApiContext> {
   async isCompatible({ embeddable }: EmbeddableApiContext) {
     const { capabilities } = this.application;
     const hasDiscoverPermissions =
-      (capabilities.discover_v2.show as boolean) || (capabilities.discover_v2.save as boolean);
+      (capabilities.discover_v2?.show as boolean) || (capabilities.discover_v2?.save as boolean);
```

Comment:
Just curious why the `discover_v2` capability is optional now? I would have thought only the ones listed in the PR would be.

Reply:
Good catch, discover_v2 is not optional, it's a leftover.  Reverting this 👍

Reply:
Should the same be true for maps_v2 then? Should those changes be reverted?

Reply:
Thanks for updating! And in that case, seems like you'll no longer need approval from Data Discovery 🙂

Reply:
@nreese tested locally and I think you're right, I removed the optional chaining - hopefully CI would tell us the truth :) Thanks!

Reply:
Thanks @davismcphee ! :)